vice/src/arch/libretro/retrostubs.c

```diff
@@ -432,7 +476,7 @@ void retro_poll_event()
     //if(SHOWKEY==-1) /* retro joypad take control over keyboard joy */
     /* override keydown, but allow keyup, to prevent key sticking during keyboard use, if held down on opening keyboard */
     /* keyup allowing most likely not needed on actual keyboard presses even though they get stuck also */
-    {
+    if (CTRLON==-1) {
```

Comment:
Why is this called CTRLON when the tab key toggles it? Shouldn’t it be called TABHELD?

Reply:
It is supposed to represent the C64 CTRL key, which is TAB. But I'm down with whatever is the most simplest to understand.

Reply:
Ah ok sounds good